include/bluetooth/services/hids_c.rst

```diff
@@ -0,0 +1,108 @@
+.. _hids_c_readme:
+
+GATT Human Interface Device (HID) Service Client
+################################################
+
+HID Service Cliemt module can be used to interact with a connected HID server.
+The instance of :ref:`hids_readme` may be used as such a server.
+The client uses the :ref:`gatt_dm_readme` module to aquire all the attribute handles required to interract with HID server.
+Some additional data is required to be read from the discovered descriptors before HID client is ready to work.
+This process is started automatically just after handles are assigned.
+When the process finishes successfully the :cpp:type:`bt_gatt_hids_c_ready_cb` function is called.
+Otherwise we may expect a call of the :cpp:type:`bt_gatt_hids_c_prep_fail_cb` function.
+
+
+Configuration parameters
+========================
+
+Apart from standard values, we have only one important additional setting here:
+
+- ``BT_GATT_HIDS_C_REPORTS_MAX`` - it sets maximum number of total repports supported by the library.
+  The report memory is shared along all the HIDS Client objects so it should be set
+  to the maximum total number of reports supported by the whole application.
+
+
+HIDS client readiness state
+===========================
+
+There are 2 functions to get the readiness state of the Service Client object:
+
+- :cpp:func:`bt_gatt_hids_c_assign_check` witch checks if :cpp:func:`bt_gatt_hids_c_handles_assign` function was already called and the post discovery process has been started.
+- :cpp:func:`bt_gatt_hids_ready_check` witch checks if the client object is fully ready to interract with HID server.
+  It is set just before the :cpp:type:`bt_gatt_hids_c_ready_cb` function is called.
```

Comment:
What is "it" here? What is set?

Reply:
The ready flag. True - it is poorly described.

Reply:
Changed to:
```
 The ready flag is set just before the :cpp:type:`bt_gatt_hids_c_ready_cb` function is called.
```